docs/getting-started-scalardl-auditor.md

```diff
@@ -0,0 +1,545 @@
+               "state": 3,
+               "nonce": "ef041eb4-f9c3-455a-9dab-0f081a01d13b"
+           },
+           "signature": "MEUCIEzeQzX3poEfM6dlgmaLIMlMI0efnsYTfY25nru1DSeXAiEAiszgxFIO3Gv4yxBI6Sy/zph1pnnwQJRuGifio09fQOg=",
+           "hash": "zqm/vXPC8o5mL5YUaAPOrIYK/qBd3kXVjHXUH7rdjnE=",
+           "prev_hash": ""
+       },
+       "error_message": null
+   }
+   ```
+   * Reference information
+       * If the asset data is not tampered with, the contract execution request (execute-contract command) returns `OK` as a result.
+       * If the asset data is tampered with (e.g. the `state` value in the DB is tampered with), the contract execution request (execute-contract command) returns a value other than `OK`  (e.g. `INCONSISTENT_STATES`) as a result, like the following.
+         ```console
+         # ./scalardl-java-client-sdk-3.4.0/bin/execute-contract --properties ./client.properties --contract-id StateReader --contract-argument '{"asset_id": "test_asset"}'
+         {
+             "status_code": "INCONSISTENT_STATES",
+             "output": null,
+             "error_message": "The results from Ledger and Auditor don't match"
+         }
+         ```
+           * In this way, the Scalar DL can detect data tampering.
+
+1. Execute validation request of the asset.
+   ```console
+   # ./scalardl-java-client-sdk-3.4.0/bin/validate-ledger --properties ./client.properties --asset-id "test_asset"
+   {
+       "status_code": "OK",
+       "Ledger": {
+           "id": "test_asset",
+           "age": 0,
+           "nonce": "ef041eb4-f9c3-455a-9dab-0f081a01d13b",
+           "hash": "zqm/vXPC8o5mL5YUaAPOrIYK/qBd3kXVjHXUH7rdjnE=",
+           "signature": "MEQCIB0lHtURoLYSz0vSpV4BhYmYBqzeSrYekN/GdP+quBLtAiBeQWMeLDEdxzeIoAkD/Cbi3zbIDud8CjK6gN/IpQKW0Q=="
+       },
+       "Auditor": {
+           "id": "test_asset",
+           "age": 0,
+           "nonce": "ef041eb4-f9c3-455a-9dab-0f081a01d13b",
+           "hash": "zqm/vXPC8o5mL5YUaAPOrIYK/qBd3kXVjHXUH7rdjnE=",
+           "signature": "MEQCIBP312FoK5pX/eSzNiJ+zCJA5PVjlbFVmMYujEESy978AiATC19Fo6Y/YGWIFgDKdfIZhLAcUcHeyNloU1YQqJBy3Q=="
+       },
+       "error_message": null
+   }
+   ```
+   * Reference information
+       * If the asset data is not tampered with, the validation request (validate-ledger command) returns `OK` as a result.
+       * If the asset data is tampered with (e.g. the `state` value in the DB is tampered with), the validation request (validate-ledger command) returns a value other than `OK` (e.g. `INVALID_OUTPUT`) as a result, like the following.
+         ```console
+         # ./scalardl-java-client-sdk-3.4.0/bin/validate-ledger --properties ./client.properties --asset-id "test_asset"
+         {
+             "status_code": "INVALID_OUTPUT",
+             "output": null,
+             "error_message": "Validation failed."
+         }
+         ```
+           * In this way, the Scalar DL can detect data tampering.
+
+## Step 8. Delete all resources
```

Comment:
Ditto

Reply:
As I described in the another comment, I want to make this guide users can use by copy and paste as much as possible.  
And, strictly, the `container names` are different that will be deleted in this section.  

So, I think it is better that we don't separate this section.  
What do you think about the above policy?

Reply:
@kota2and3kan san,
It is easy to use, but there is some redundancy.
So, you can confirm with @feeblefakie san.

Reply:
As I mentioned in another comment, I will keep this redundancy in this PR.